src/entities/common/closeable-window-entity.ts

```
@@ -57,7 +57,7 @@ export class CloseableWindowObject extends BaseTappableGameObject {
 
   public close(): void {
     if (this.opened === false) {
-      return console.warn("CloseableWindowObject is already closed");
+      return console.warn("CloseableWindowEntity is already closed");
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)